README.md

```diff
@@ -1,127 +1,194 @@
+|      ✅       | `test:performance` | Runs the performance test suite.                                                                                                                                       |
 
 [comment]: <> (The following scripts are not to be used except for CI/CD environments)
-[comment]: <> (| ❌            | `deploy`            | Runs `build` script, then pushes the output to GH Pages.                                      |)
-[comment]: <> (| ❌            | `deploy:clean`      | Runs `build` script, clears `gh-pages` branch, then pushes the output to GH Pages.            |)
-[comment]: <> (| ❌            | `deploy:folder`     | Runs `build` script, then pushes the output to the specified folder in GH Pages.              |)
+[comment]: <> (| ❌ | `deploy` | Runs `build` script, then pushes the output to GH Pages. |)
+[comment]: <> (| ❌ | `deploy:clean` | Runs `build` script, clears `gh-pages` branch, then pushes the output to GH Pages. |)
+[comment]: <> (| ❌ | `deploy:folder` | Runs `build` script, then pushes the output to the specified folder in GH Pages. |)
 
 **Please follow the README of each package you intend to work with on how to get set up and their custom scripts.** However, the above scripts can be run from the root directory in the following manner.
 
 ### Package names
+
 Each package is named with the `@deriv/` prefix, however for the scripts above, you do not need to add the `@deriv/` prefix as the scripts already prefix the 1st argument of the script with `@deriv/`. **However**, if you do use the `lerna` CLI directly, then you will need to use the full package name including the `@deriv/` prefix.
 
-You can find the names of packages by first navigating to the `packages` folder. Each subfolder is a package, and contains a `package.json` file. The value of the `name` key in `package.json` is the package name.
+You can find the names of packages by first navigating to the `packages` folder. Each sub-folder is a package and contains a `package.json` file. The value of the `name` key in `package.json` is the package name.
 
 ### Usage
+
 #### Configuring Hosts file
 
 In order to run our solution for the first time, you need to configure your `hosts` file:
+
 1. Open terminal.
 2. Open `hosts` file in your preferred text editor, f.e `sudo vim /etc/hosts`.
-3. Add a new entry pointing to `127.0.0.1     localhost.binary.sx`.
+3. Add a new entry pointing to `127.0.0.1 localhost.binary.sx`.
 4. Save the file and proceed to the next step.
 
 #### Starting a Development Server
 
 If you wish to work on Core, simply run `npm run serve core`.
 
 But for working on any of the other packages (such as Trader, Bot, P2P), perform the following:
+
 1. Open 2 terminals.
 2. Run `npm run serve {package name}` in the first one. e.g.: `npm run serve translations`, `npm run serve bot`, etc.
 3. Then run `npm run serve core` in the second one.
 
 #### How to Clean Packages
+
```

Comment:
in some cases this script now working. And we should run it using sudo or maybe regain file premissions using `chown` command to make it work, What do you think about it?